source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your local setup:
+
+In [govuk_app_config], add your local domain to the allowed CORS domains:
+
+```
+  GOVUK_DOMAINS = [
+    '*.publishing.service.gov.uk',
+    # "*.#{ENV['GOVUK_APP_DOMAIN_EXTERNAL'] || ENV['GOVUK_APP_DOMAIN'] || 'dev.gov.uk'}"
+     '*.dev.gov.uk'
+  ].uniq.freeze
+```
+
+You need to change the frontend app you are working on to use the modified [govuk_app_config]. For example in [frontend] update the Gemfile:
+
+```ruyby
+...
+gem 'govuk_app_config', path: '/var/govuk/govuk_app_config'
+...
+gem 'govuk_publishing_components', path: '/var/govuk/govuk_publishing_components'
+...
+```
+
+and set in `development.rb`:
+
+```ruby
+...
+config.assets.debug = false
+...
+```
+
+Update the Gemfile in [static] to point to [govuk_publishing_components] gem:
+
+```ruby
+...
+gem 'govuk_publishing_components', path: '/var/govuk/govuk_publishing_components'
+...
+```
+
+Run up static in the vm:
+
+```shell
+$dev cd /var/govuk/static
+$dev govuk_setenv static ./startup.sh --live
+```
+
+Run up frontend in the vm:
```

Comment:
Using 'frontend' as an example could be confusing for new starters who may assume we only have one frontend app called Frontend. Probably worth making clear that this is an example, but the instructions should apply to other frontend apps too (I think there may be a list of Frontend apps somewhere)